packages/core/core/src/Transformation.js

```diff
@@ -549,17 +555,17 @@ export default class Transformation {
       return null;
     }
 
-    let cached = await this.options.cache.get<{|assets: Array<AssetValue>|}>(
-      cacheKey,
-    );
+    let cached = await this.options.cache.get<{|
+      assets: Array<[AssetValue, Array<DependencyValue>]>,
```

Comment:
Would it be easier to split the AssetValue type into two: one with dependencies as before, and one without? Then most of the code could keep using the existing one, and only the one in the graph would omit it.